.pre-commit-config.yaml

```diff
@@ -0,0 +1,44 @@
+repos:
```

Comment:
This is all great - I'm not going to dig into the specifics of the code formatting choices here, I trust they're reasonable. It's more important that we've made a choice and that it's enforced than what that choice actually is.